packages/graphql/tests/tck/federation/authorization.test.ts

```diff
@@ -129,9 +129,8 @@ describe("Federation and authorization", () => {
 
         expect(formatCypher(result.cypher)).toMatchInlineSnapshot(`
             "MATCH (this:User)
-            WHERE this.id = $param0
             WITH *
-            WHERE ($isAuthenticated = true AND ($jwt.sub IS NOT NULL AND this.id = $jwt.sub))
+            WHERE (this.id = $param0 AND ($isAuthenticated = true AND ($jwt.sub IS NOT NULL AND this.id = $jwt.sub)))
```

Comment:
Is this a bug fix because of moving to the translation layer?

Reply:
It's because in the new translation layer, we treat the authorization predicate as a filter and we apply all the filters in the same place, while before you had translate-top-level-match that was producing a match plus an entity filter, and then create-projection-and-params that was producing the projection plus the authorization filters. The behavior is the same!